### ISSUE TITLE
Extra Features

### ISSUE THREAD

**Issue by SavJas** [simen64/Discord-RAT-v2#1]
Hello, I tested your discord RAT V2 and it was nice. Even tho there was some problems with file commands, it was okay. I am writing in order to encourage you to add to commands. Firstly !history, grabs web history from chrome, !passwords, grabs chrome saved passwords, with a nice looking embed. Hope you see that and make it even better

**Comment by simen64** [comment]
Hi, thanks for taking a look at it! Regarding the commands. i havent been able to work a lot on the project as of recently, but ill see if i find some time. The functions of the commands you mentioned have been made in other forms many times already so maybe ill just implement someone else's code (giving credit ofc.) Could you also tell me what the issues with the file commands were? And what OS you are running.

**Comment by SavJas** [comment]
> Hi, thanks for taking a look at it! Regarding the commands. i havent been able to work a lot on the project as of recently, but ill see if i find some time. The functions of the commands you mentioned have been made in other forms many times already so maybe ill just implement someone else's code (giving credit ofc.) Could you also tell me what the issues with the file commands were? And what OS you are running.

I am running Windows 11. Lets say I type !cd Desktop or Downloads. It says Could not find directory: Desktop/Downloads. Also for the !ls, I type !ls Downloads but it lists the folder that the python file Rat-V2 is. The upload, delete and download command work great. But the !ls & !cd, does not work for me. Is it me or my os, I don't know! Check: 
![image](https://github.com/simen64/Discord-RAT-v2/assets/86425637/ee15359e-e591-4b5a-b71f-9017274be394)

**Comment by SavJas** [comment]
Also, if you can/will update it, make it on first run to go into Startups Apps, in order to run the program every time the user turns on the computer.

**Comment by simen64** [comment]
Imma look into the cd issue, as it is mostly tested on Linux it may be a windows quirk. And the ls command doesnt take an argument path, but i probably should so i can add that aswell. When it comes to startup i have decided not to make it automatically do it, as this lets the deployment more flexible, you can see this in my HID-scripts repo. But i can add a command for it.

**Comment by SavJas** [comment]
> Imma look into the cd issue, as it is mostly tested on Linux it may be a windows quirk. And the ls command doesnt take an argument path, but i probably should so i can add that aswell. When it comes to startup i have decided not to make it automatically do it, as this lets the deployment more flexible, you can see this in my HID-scripts repo. But i can add a command for it.

![image](https://github.com/simen64/Discord-RAT-v2/assets/86425637/f339b993-7a17-4fd9-9060-de5d5cd37853)
Would you add these too?

**Comment by simen64** [comment]
Those are things i want to add yes, but i havent had time as i work on a lot of projects. I also want to in the future learn C and make a v3 version in C.

**Comment by SavJas** [comment]
> Those are things i want to add yes, but i havent had time as i work on a lot of projects. I also want to in the future learn C and make a v3 version in C.

Yeah that would be nice, I finished with HTML, CSS and JS and now I learning python.
